steps/configcat-feature-flag-sync/0.0.2/step.yml

```diff
@@ -0,0 +1,68 @@
+title: ConfigCat Feature Flag Sync
+summary: Scans the source code for feature flags and uploads the references to the
+  ConfigCat Dashboard
+description: "This Step helps you get rid of technical debt by scanning the source
```

Comment:
At first it was a but hard to read this description because of the formatting. The newlines were included as characters which did not help in separating the different sections. 

In the yaml file can actually use the `|-` specifier which will preserve the newlines in your formatted text. I would recommend using that (we are also using it in our step descriptions).

Here is an example: https://github.com/bitrise-steplib/steps-xcode-archive/blob/16729ba49389a2368689f6e6e6b8503cc6d394ed/step.yml#L3

Reply:
In the original source, we are using `|-`:
https://github.com/configcat/bitrise-step-configcat-feature-flag-sync/blob/0.0.2/step.yml#L4

I think the `bitrise run share-this-step` changes the step.yml format to this.